python/tvm/relay/backend/contrib/ethosu/tir_to_cs_translator.py

```diff
@@ -106,21 +202,31 @@ def translate(tir_module, params):
         base addresses to be used by the driver
     """
 
+    # The NPU has 6 usable regions ranging from 0-6
+    # The regions 0, 3, and 4 is already used for input,
+    # output and constant, respectively (See _get_regions()).
+    # Thus, for scratch we are left with 5, 2 and 1.
+    candidate_regions_for_scratch = [5, 2, 1]
+    (
+        scratch_region_map,
+        tvmbaw_workspace_size,
+        tvmbaw_region,
```

Comment:
Similar here, TVMBAW is a detail of how Allocates are lowered

Reply:
happy to align with the name we end up deciding above...